ci-operator/step-registry/azure/deprovision/service-principal-and-custom-role/azure-deprovision-service-principal-and-custom-role-ref.metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+	"path": "azure/deprovision/service-principal-and-custom-role/azure-deprovision-service-principal-and-custom-role-ref.yaml",
```

Comment:
how about rename `service-principal-and-custom-role` to `sp-and-custom-role` for a shorter name.